frontend/src/components/Table/VirkarekisteriTable.tsx

```diff
@@ -0,0 +1,175 @@
+/* eslint-disable react/jsx-key */
+import {
+  TableContainer,
+  Box,
+  Table,
+  TableHead,
+  TableRow,
+  TableCell,
+  TableBody,
+  TableFooter,
+  alpha,
+} from '@mui/material';
+import React from 'react';
+import type { Column } from 'react-table';
+import { useTable, useSortBy } from 'react-table';
+
+// Define the data type
+interface Data {
```

Comment:
These will all live in Redux, as I'm sure you're aware. 
It's fine like this as an example.